src/components/Dialogs/TransactionDialog/TransactionDialog.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import ActionDialog, { ActionDialogProps } from '../ActionDialog/ActionDialog'
+import { TextContainer, StyledTransactionIllustration, Spinner } from './TransactionDialog.style'
+import { StyledTitleText, StyledDescriptionText } from '../MessageDialog/MessageDialog.style'
+
+export type TransactionDialogProps = {
+  title: string
+  description: string
```

Comment:
I think we will always use this dialog in the same context, let's define those right here so we don't need to copy-paste the same thing

Reply:
@mikkio-j This wasn't resolved - those values shouldn't be passed as props, but defined here in the component